blivetgui/exception_handler.py

```diff
@@ -67,14 +68,15 @@ def handle_exception(self, exc_type, exc_value, exc_traceback):
         else:
             tr_str = "".join(traceback.format_tb(exc_traceback))
 
-        if self.allow_ignore:
+        allow_report = command_exists("gnome-abrt")
```

Comment:
What about reporting exceptions when running Anaconda ? Is that also handled by this or not yet/elsewhere ?